app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -91,6 +92,20 @@ def localized_locations(locations)
         end
       end
 
+      def scrub_message(message)
+        filtered_message = message.dup
+        filtered_message.gsub!(/sponsorID \d+/i, 'sponsorID [FILTERED]')
+        filtered_message
```

Comment:
Rather than cloning and mutating the string in-place, we could use the normal form of `gsub` which returns a copy of the original string?

```suggestion
        filtered_message.gsub(/sponsorID \d+/i, 'sponsorID [FILTERED]')
```

Reply:
Good point!  I made the change. 👍🏻